packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts

```diff
@@ -25,16 +35,16 @@ async function convertSvgToReactComponent(
 
   await fs.writeFile(outputPath, jsCode)
 
+  console.log()
   console.log(`SVG converted to React component: ${outputPath}`)
 }
 
 export default async function transform(file: FileInfo, api: API) {
   const j = api.jscodeshift
-
   const root = j(file.source)
 
-  // Do this lazily
```

Comment:
The cost of importing `getPaths` has already been paid by `replaceComponentSvgs.yargs.ts`, so moved this to a regular import, so you don't have to wonder why we're doing a lazy import.